pkg/controller/realmadmin/settings_modify.go

```diff
@@ -233,7 +233,8 @@ func (c *Controller) HandleSettings() http.Handler {
 			if c.config.Features.EnableUserReport && form.AllowUserReport {
 				currentRealm.AddUserReportToAllowedTestTypes()
 				currentRealm.AllowAdminUserReport = form.AllowAdminUserReport
-			} else {
+			}
+			if !c.config.Features.EnableUserReport {
```

Comment:
This will have the (potentially intended) side-effect of disabling user-report in the following scenario:

1. Enabled at system level
2. Realm enables
3. Disabled at system level
4. Realm updates something else on that page

Is that intended?

Reply:
good catch - fixed for what I intended it to be.